cli_tools_tests/common/config/config.go

```diff
@@ -0,0 +1,36 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
@zoran15  from #1379 

With the new folder structure, i.e.:
cli_tools_tests/module
cli_tools_tests/e2e

Where each subfolder holds certain category of tests, common seems out of place, as it could be mistaken for 'common' tests, not common code shared between different test categories.

Is there overlap between common code in e2e and module tests? If not, we can split the code and move it into e2e/common and module/common. If there is an overlap, renaming common to indicate it doesn't actually hold tests is an option.

Reply:
Updated naming in current version:

```
cli_tools_tests
├── common
├── e2e_tests
└── module_tests
```

Regarding common code, some is specific to Tony's custom junitxml work, and that wouldn't be used in `module_tests`. But there are some utilities for creating GCP resources that are agnostic to the testing suite.

Reply:
Since we are moving folders around, maybe we can take a step back and think where this folder should naturally go. It doesn't contain actual tests, but testing framework. Maybe it should be a package of its own? Maybe put it in test-infra?

Reply:
Talked offline -- created a new root directory for common code